web/src/main/java/se/inera/intyg/intygstjanst/web/integration/SendCertificateToRecipientResponderImpl.java

```diff
@@ -64,28 +56,29 @@ public SendCertificateToRecipientResponseType sendCertificateToRecipient(
 
         SendCertificateToRecipientResponseType response = new SendCertificateToRecipientResponseType();
 
-        final String mottagareId = request.getMottagare().getCode();
-        final String intygsId = request.getIntygsId().getExtension();
+        final var mottagareId = request.getMottagare().getCode();
+        final var intygsId = request.getIntygsId().getExtension();
+        final var hosPersonal = request.getSkickatAv().getHosPersonal();
```

Comment:
@mww-sopra-steria Is it okay that I only take the hosPersonal from SkickatAv or do I need to save the personId also? I'm thinking that the personId should always be the same as the patientPersonId

Reply:
I think it is OK! When "act on behalf"-functionality is added to Minaintyg, it will only be sent through the new internal rest api and we will then cater for it there. So keep the implementation here as you written it.